R/metab.bookkeep.R

```diff
@@ -73,7 +73,7 @@ metab.bookkeep <- function(do.obs, do.sat, k.gas, z.mix, irr, ...){
 		warning("datetime not found, inferring sampling frequency from # of observations") # issue a warning (note checks in addNAs)
 		# NOTE: because of the checks in addNA's, it is unlikely a user would receive this warning via metab()
 		# warning will only be seen through direct use of metab.bookkeep when datettime is not supplied
-		freq <- nobs
+		freq <- nobs - 1
```

Comment:
I _have_ switched to midnight to midnight side because if you apply a mass balance algebraic approach to O2 data (e.g. bookkeeping), then you will be missing a portion of the mass balance for O2 in a day using just a 1 midnight. Maybe less important for non-algebraic approaches.